google-cloud-bigtable/lib/google/cloud/bigtable/instance.rb

```diff
@@ -485,7 +485,7 @@ def tables
         #   # Default view is full view
         #   table = instance.table("my-table", perform_lookup: true)
         #   puts table.name
-        #   puts table.column_families
+        #   puts table.column_families.inspect
```

Comment:
Why is `#inspect` being added to these lines?

Reply:
Thanks for catching that. Removed.